galley/pkg/config/analysis/msg/messages.yaml

```
@@ -243,3 +243,13 @@ messages:
     description: "The MeshPolicy resource is deprecated and will be removed in a future Istio release. Migrate to the PeerAuthentication resource."
     template: "The MeshPolicy resource is deprecated and will be removed in a future Istio release. Migrate to the PeerAuthentication resource."
 
+  - name: "InvalidRegexp"
+    code: IST0122
+    level: Warning
+    description: "Invalid Regex"
+    template: "Invalid or unsupported %s regexp %q."
```

nit: I'd suggest something slightly more explicit:

```suggestion
    template: "Field '%s' has a regular expression with invalid syntax: '%q'."
```

Honestly I'm not in love with my suggested wording, but at least having quotes around each argument helps in my opinion